content/blog/2020-09-28-september-20-community-gems.md

```diff
@@ -0,0 +1,198 @@
+---
+title: September '20 Community Gems
+date: 2020-09-28
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  customizing your DVC plots, the difference between external dependencies
+  and outputs, and how to save models and data in continuous integration.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  customizing your DVC plots, the difference between external dependencies
+  and outputs, and how to save models and data in continuous integration.
+picture: 2020-09-28/Gems_Sept_20.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/september-20-community-gems/512
+tags:
+  - Discord
+  - Gems
+  - CML
+  - Hyperparameters
+  - External
```

Comment:
External Data? or do we have any similar tags already?

Reply:
We have neither, and nothing similar, so it's up to us. What do you prefer?